### PR TITLE
feat: add journey aid posthog events

### PR DESCRIPTION
**Pull request by rosvik**
closes https://github.com/AtB-AS/kundevendt/issues/19521

### CODE REVIEW

File: src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_ProfileStack/Profile_TravelAidScreen.tsx [hunk 1]
```diff
@@ -52,9 +56,17 @@ export const Profile_TravelAidScreen = () => {
...
-            onValueChange={(checked) =>
-              setPreference({journeyAidEnabled: checked})
-            }
+            onValueChange={(checked) => {
+              analytics.logEvent(
+                'Journey aid',
+                'Journey aid preference toggled',
+                {
+                  enabled: checked,
+                  isScreenReaderEnabled,
```
Comment (tormoseng) [inline src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_ProfileStack/Profile_TravelAidScreen.tsx:65]: You don't need to add `screenreader: isScreenReaderEnabled` to have a parameter name on the event?

Comment (rosvik) [reply src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_ProfileStack/Profile_TravelAidScreen.tsx]: This is shorthand for `isScreenReaderEnabled: isScreenReaderEnabled` in javascript. But I did want to consistently use `screenReaderEnabled` here, so thanks for making me notice 😄

Comment (rosvik) [reply src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_ProfileStack/Profile_TravelAidScreen.tsx]: So from now, it should be `screenReaderEnabled: true/false` in posthog

**Review by tormoseng — APPROVED** [review]
LGTM

**Pull request merged**